src/lib/component/partial/EventFiltersList/EventFiltersListItem.tsx

```diff
@@ -0,0 +1,75 @@
+import React from "/vendor/react";
+import { TableRow } from "/vendor/@material-ui/core";
+
+import { createStyledComponent, NamespaceLink } from "/lib/component/util";
+import ResourceDetails from "/lib/component/partial/ResourceDetails";
+import TableOverflowCell from "/lib/component/partial/TableOverflowCell";
+
+const AllowLabel = (createStyledComponent as any)({
+  name: "EventFilterListItem.AllowLabel",
+  component: "span",
+  styles: (theme: any) => ({
+    color: "white",
+    fontSize: "0.7rem",
+    fontWeight: "bold",
+    backgroundColor: theme.palette.success,
+    padding: theme.spacing.unit / 2,
+  }),
+});
+
+const DenyLabel = (createStyledComponent as any)({
```

Comment:
A nitpick:
I haven't really used this method of creating a styled component (usually just worked with applying styles to ones I created a different way), but I wonder if it would be better to create a single label, and then apply an additional class which only has the appropriate background colour if needed? That way it's a little more dry.

Reply:
Yeah, that's a good point -- I would usually opt for creating a base `Label`, so that it can used elsewhere, but these labels felt bespoke and specifically tied to the `EventFilterListItem` and not necessarily something we have the intention of using elsewhere. Though, an argument can be made for both sides. 

I'm easy either way. 

@jamesdphillips, do you have an opinion here?

Reply:
A base label component would make sense to me but I think in the short term this is fine. That is until we start adding more labels / using them elsewhere in the application. 👌

Reply:
Thanks! I think I might want/need to turn the Label into a base component for the Event Filter Details View, but I'll cross that bridge in that PR.

Reply:
After the long weekend I feel extra late to this conversation. I didn't mean have it as it's own entire component. An example of what I meant might be `base/banner.js`. It has a base style, and then each action `success`, `error`, etc. applies a different class which just has a background property. 

I know this is merged, and I am totally fine with what was merged, but I just wanted to clarify what I meant.